sycl/doc/design/images/L0_UR_command-buffer.svg

```diff
@@ -1,17 +0,0 @@
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 731 231" width="731" height="231">
```

Comment:
What's the reason of changing svg to jpg? I think it's better to keep svg and remove jpg

Reply:
Good question, we changed the tool used to generate the image and just went with the default exported format. But it can export to svg as well, so updated the image back to svg format

Reply:
The svg file looks invalid: for some reason it's actually a binary file.

Reply:
Fixed, GitHub looks to be rendering the diff of the SVG file okay now